calyx-py/calyx/builder.py

```diff
@@ -319,6 +312,10 @@ def and_(self, name: str, size: int) -> CellBuilder:
         """Generate a StdAnd cell."""
         return self.cell(name, ast.Stdlib.op("and", size, False))
 
+    def not_(self, name: str, size: int) -> CellBuilder:
+        """Generate a StdNot cell."""
+        return self.cell(name, ast.Stdlib.op("not", size, False))
```

Comment:
Solid! Long term, we could consider overriding the `!` operator for the calyx Port class to construct this for you